data/recipes.yaml

```diff
@@ -13,6 +13,21 @@
   out:
   - [4, board]
 
+- in:
+  - [1, log]
+  - [8, water]
+  out:
+  - [8, paper]
+
+- in:
+  - [256, paper]
+  - [1, LaTeX]
+  required:
+  - [1, big furnace]
+  out:
+  - [1, PhD thesis]
+  time: 65536
```

Comment:
Hah, 2h of in-game time sounds fine, but I see some moral problems with robots passing around finished thesis instead of properly installing it on themselves. :smile:

Reply:
Hah!  Well, you can't expect robots to understand the finer points of morality...

When testing it, I sped the game up to 1024 tick/s so it only took about 60s... :smile:

Reply:
I just put it in as a joke, I wanted to add `paper` in order to make a `dictionary`, then I started thinking what else you could do with `paper` and `LaTeX`...